### PR TITLE
Assembly builder append names

### PR DESCRIPTION
**Pull request by Qining**

### CODE REVIEW

File: test/opt/assembly_builder.h [hunk 1]
```diff
@@ -111,6 +111,15 @@ class AssemblyBuilder {
     });
   }
 
+  AssemblyBuilder& AppendNames(const std::vector<std::string>& vec_asm_code) {
```
Comment (antiagainst) [inline test/opt/assembly_builder.h]: blurb

Comment (Qining) [inline test/opt/assembly_builder.h]: added in 9783e0a

File: test/opt/assembly_builder.h [hunk 2]
```diff
@@ -111,6 +111,18 @@ class AssemblyBuilder {
     });
   }
 
+  // Appends OpName instructions to this builder. Instrcution strings that do
```
Comment (dneto0) [inline test/opt/assembly_builder.h]: "Instruction"

**Comment by dneto0** [comment]
+2

**Comment by dneto0** [comment]
Rebased and pushed onto master as ecb5692389

**Pull request closed without merging**